website/docs/cloud-docs/api-docs/variable-sets.mdx

```diff
@@ -309,6 +309,15 @@ List all variable sets for a workspace.
 | --------------- | ---------------- |
 | `:workspace_id` | The workspace ID |
 
+### Query Parameters
+
+Both of the endpoints above support pagination [with standard URL query parameters](/cloud-docs/api-docs/#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
```

Comment:
Suggest rewording to "Both of the endpoints support pagination [with standard URL query parameters](/cloud-docs/api-docs/#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling does not automatically encode URLs."